docs/index.md

```diff
@@ -69,20 +69,6 @@ If you are a **developer** with EDA background, learn more about how you
 can use OpenROAD as the infrastructure for your tools in the
 [Developer Guide](contrib/DeveloperGuide.md) section.
 
-## How to get in touch
-
-We maintain the following channels for communication:
-
--   Project homepage and news: <https://theopenroadproject.org>
--   Twitter: <https://twitter.com/OpenROAD_EDA>
--   Issues and bugs:
-    -   OpenROAD: <https://github.com/The-OpenROAD-Project/OpenROAD/issues>
--   Discussions:
-    -   OpenROAD: <https://github.com/The-OpenROAD-Project/OpenROAD/discussions>
--   Inquiries: openroad@eng.ucsd.edu
-
-See also our [FAQs](user/FAQS.md).
-
```

Comment:
Why remove this?

Reply:
Well I thought the main communication channels are in the icons already, the github issues/discussions are available from github link so it might be redundant information.

Reply:
I think, in this case, the redundancy is good. The icons are small and might not be noticed by users, while the text info will be easier to find if someone looks for -- including in searches.

Reply:
Ok, will revert.